### ISSUE TITLE
what's the meaning of the name of json files?

### ISSUE THREAD

**Issue by Moran232** [SEAL-UCSB/block_compression#4]
Hi, I noticed there are many *.json files in the ACCURACY. Can you explain the meanings of the figures like '128x16.dc1'? and what does the 'dc1' stand for?
And, the you only pruned the VGG16，VGG16_bn and AlexNet? How about the LeNet5 at MNIST and VGG14 @ cifar10 according to your paper ?

**Comment by ji-u** [comment]
Sorry, those files are configuration files used during my experiments. They are not all final data appeared in the paper. And I have deleted those files in current master brunch many days ago.
'dc1' means I use Deep Compression's pruning rate minus 10% for each layer.

Code for MNIST and CIFAR are written by @liangling76 . He uses the same compress.py file and implement the training code for those two models. It is easy to reproduce the result with compress.py in accuracy folder.

**Comment by Moran232** [comment]
thank you for your reply~
